File: sdk/samples/FrameworkSamples/CFX/Serialization/Xml Serialization/IXmlSerializable/cs/service/app_code/service.asmx.cs
using System;
using System.Collections;
using System.ComponentModel;
using System.Data;
using System.Diagnostics;
using System.Web;
using System.Web.Services;
/// <summary>
/// Summary description for Service_asmx.
/// </summary>
/// 

namespace Microsoft.Samples
{
	[WebService(Namespace = "http://demos")]
	public class Service_asmx
	{
		[WebMethod]
		public Order GetOrder()
		{
			Order order = Order.Create(5);
			return order;
		}
	}
}

File: sdk/samples/FrameworkSamples/CFX/Serialization/Xml Serialization/IXmlSerializable/vb/service/app_code/service.asmx.vb
Imports System.Web.Services
Imports System.Web.Services.Protocols
Imports Microsoft.Samples

<WebServiceBinding(ConformanceClaims:=WsiClaims.BP10, EmitConformanceClaims:=True)> _
Public Class Service_asmx

    <WebMethod()> _
    Public Function GetOrder() As Order
        Dim order As Order = New Order()
        order.Fill(5)
        Return order
    End Function

End Class
